applications/lib_app/rsa_util/rsa_util.cpp

```diff
@@ -252,3 +252,42 @@ bool openssl_gen_dh(const char* dh_key, int bitsize)
     return true;
 #endif
 }
+
+bool get_server_name(const char *server_crt_path, char *server_name)
+{
+    char temp[256];
+
+    FILE *f = fopen(server_crt_path, "r");
+    if (f == NULL) {
+        return false;
+    }
+
+    while (fgets(temp, sizeof(temp), f) > 0) {
```

Comment:
You can use a command line to get subject: openssl x509 -noout -subject -in server.crt